.dockerignore

```diff
@@ -0,0 +1 @@
+.github
```

Comment:
Maybe `.git` as well? Not important, just curious.

Reply:
`.git` isn't that big (.github can be massive), but we can add it.